### PR TITLE
Add documentation for Consensus Commit configurations

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Scalar DB design document
+# Scalar DB design document
```
Comment (brfrn169) [inline docs/design.md:1]: This PR includes some refactoring things for the existing documents.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

File: docs/configurations-for-consensus-commit.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+The basic configurations for Consensus Commit is as follows:
+
+```properties
+# Default isolation level for ConsensusCommit. Either SNAPSHOT or SERIALIZABLE can be specified. SNAPSHOT is used by default.
+scalar.db.consensus_commit.isolation_level=SNAPSHOT
+
+# Default serializable strategy for ConsensusCommit transaction manager.
+# Either EXTRA_READ or EXTRA_WRITE can be specified. EXTRA_READ is used by default.
+# If SNAPSHOT is specified, this is ignored.
+scalar.db.consensus_commit.serializable_strategy=EXTRA_READ
```
Comment (feeblefakie) [inline docs/configurations-for-consensus-commit.md:23]: The comment explains but it’s a bit confusing to see both SNAPSHOT and EXTRA_READ are used in the configuration.
Should it explain each configuration separately?
How about making it a table-format like this? (can be simpler than this, though)
https://kafka.apache.org/documentation/#brokerconfigs

**Review by feeblefakie — COMMENTED** [review]
The contents are looking good! Thank you!
I left a suggestion about the format. PTAL!

File: docs/configurations-for-consensus-commit.md (hunk 2 above)
Comment (brfrn169) [reply docs/configurations-for-consensus-commit.md]: I fixed it in the latest commit based on your feedback. Please take a look again when you have time!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**